content/starter-configs/tutorials/wordpress.md

```diff
@@ -96,14 +127,32 @@ services:
         # Use the tugboat-specific wp-config.local.php
         - cp "${TUGBOAT_ROOT}/.tugboat/wp-config.local.php" "${DOCROOT}/"
 
-        # Link the document root to the expected path. This example links /web
-        # to the docroot
-        - ln -snf "${TUGBOAT_ROOT}/web" "${DOCROOT}"
+        ## STOP HERE! Define your Wordpress Core docroot.
+        ## Uncomment line in either Option 1 or Option 2, depending on your setup.
+
+        # OPTION 1: Wordpress Core is at the repo root.
+        # - ln -snf "${TUGBOAT_ROOT}" "${DOCROOT}"
+
+        # OPTION 2: Wordpress Core lives in a subdirectory (in this example, 'wp-core').
+        # - ln -snf "${TUGBOAT_ROOT}/wp-core" "${DOCROOT}"
```

Comment:
Just a recommendation: change to something like
```
- ln -snf "${TUGBOAT_ROOT}/docroot" "${DOCROOT}"
```

docroot, public, web, that would make it a little more relatable as chances are `wp-core` isn't entirely correct